content/en/dashboards/widgets/slo.md

```diff
@@ -21,43 +21,48 @@ further_reading:
-3. Select up to three different time windows.
+2. Set up to three different time windows.
+3. Select your display preferences.
 
 ### Options
 
-#### Set the time window
+#### Set the time windows
 
-Select up to three different rolling time windows. Optionally, you can select a calendar time window or select **Global Time**.
-
-`Global Time` allows you to display your SLO's status and error budget over arbitrary time periods within the past 90 days. 
-
-You can specify an optional unique SLO target for the arbitrary time period. To display an error budget and to color code the SLO status value as green or red, you need to specify an SLO target. 
-
-**Note**: If the SLO input target is not specified, only the SLO status is shown and the font color remains gray.
+Select up to three different time windows from the following:
+- **Rolling time windows**: 7, 30, or 90 days
+- **Calendar time windows**: week to date, previous week, month to date, previous month
+- **Global time** (supported for metric-based and Time Slice SLOs): This option allows you to display your SLO's status and error budget over arbitrary time periods within the past 90 days. 
+  
+  **Note:** With `Global time`, you can also specify an optional unique SLO target. To display an error budget and color code the SLO status value as green or red, you need to specify an SLO target. If the SLO input target is not specified, only the SLO status is shown and the font color remains gray.
 
 #### Display preferences
 
-Select whether to show or hide remaining error budget by toggling the `Show error budget` option. If you are visualizing a monitor-based SLO with multiple groups or multiple monitors, select your `View mode`:
+Select whether to show or hide remaining error budget by toggling the `Show error budget` option. 
 
-- For monitor-based SLOs configured with a single monitor broken into multiple groups, there are the following three view modes:
+If you are visualizing an SLO with multiple groups or a monitor-based SLO with multiple monitors, select your `View mode`:
+
+- For SLOs with groups (metric-based or Time Slice SLO with groups, or monitor-based SLOs with a single monitor broken into groups), there are the following three view modes:
   - `Overall`: displays the overall SLO status percentages and targets
   - `Groups`: displays a table of status percentages for each group
   - `Both`: displays both the overall SLO status percentages and targets and table of status percentages for each group
 
 - For monitor-based SLOs configured with multiple monitors, there are the following three view modes:
   - `Overall`: displays the overall SLO status percentages and targets
-  - `Groups`: displays a table of status percentages for each group
+  - `Groups`: displays a table of status percentages for each monitor
```

Comment:
```suggestion
  - `Monitors`: displays a table of status percentages for each monitor
```
Looks like an error in our existing documentation. For multi-monitor SLOs, this option is named 'Monitors'